packages/components/src/components/checkbox/checkbox.jsx

```diff
@@ -3,7 +3,7 @@ import PropTypes  from 'prop-types';
 import React      from 'react';
 import Icon       from '../icon';
 
-class Checkbox extends React.PureComponent {
+class Checkbox extends React.Component {
```

Comment:
Any special reason for this to change into normal component and not pure one? :D

Reply:
added `getDerivedStateFromProps` with this, it doesn't work on Pure :D